### ISSUE TITLE
[a11y]: ContentSwitcher [all variants] - tabs do not have any content - need clarity

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon#13981]
### Package

@carbon/react

### Browser
Safari & FF & iPhone

### Package version

carbon/react@1.30.0

### Automated testing tool and ruleset
Manual QA

### Assistive technology

JAWS 

### Description

all the tabs

The tabs do not have any content

### WCAG 2.1 Violation
n/a

### Reproduction/example
Record `Number 51` in the below table:
https://airtable.com/shrFsaeHhImOkuVNR

### Steps to reproduce

It will be better if the content is added for each tab in A tab panel. It will make the testing complete.

This needs to be confirmed / refined by the team

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems
